frontend/model/state.js

```diff
@@ -82,10 +82,13 @@ const initialState = {
   loggedIn: false, // false | { username: string, identityContractID: string }
   namespaceLookups: Object.create(null), // { [username]: sbp('namespace/lookup') }
   reverseNamespaceLookups: Object.create(null), // { [contractID]: username }
-  periodicNotificationAlreadyFiredMap: {}, // { notificationKey: boolean },
   contractSigningKeys: Object.create(null),
   lastLoggedIn: {}, // Group last logged in information
-  preferences: {}
+  preferences: {},
+  periodicNotificationAlreadyFiredMap: {
+    alreadyFired: Object.create(null), // { notificationKey: boolean },
+    lastRun: Object.create(null) // { notificationKey: number },
+  }
```

Comment:
I think it's a general rule that whenever this initial dataset is modified it requires a corresponding section under `postUpgradeVerification` for older clients that did not start out with this data.